content/resources/help-centre/software-tips/package-manager.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Package manager
+---
+
+## What is a package manager?
+
+Much of the software that is mentioned on this website includes package manager
+installation instructions, if they are available. But what is a package manager?
+Essentially, it is a piece of software that lets us easily and conveniently
+download and install applications (known as packages), and keep those packages
+up to date. A package manager runs in the [command line]({{% ref
```

Comment:
not strictly true; windows store is a package manager (and the new `winget` actually uses the same backend as the store) and various linux package managers have graphical UIs (like snap on linux)